source/nodejs/adaptivecards-site/pages/_posts/2020/Community-Call-June.md

```diff
@@ -0,0 +1,81 @@
+---
+title: 📣 Community Call - June, 2020 
+subtitle: Build 2020 recap and a new SharePoint web part
+date: 2020-06-11
+featured_image: https://developer.microsoft.com/en-us/office/blogs/wp-content/uploads/2020/06/2-2-767x431.png
+github_username: matthidinger
+twitter: matthidinger
+---
+
+Welcome to our Adaptive Cards community call series for 2020, the 2nd Thursday of each month. We are excited to provide you with a monthly opportunity to learn about Adaptive Cards in general, how they could be a great tool in your applications and contribute to the evolution of the format and SDKs. Each month, we'll discuss new features we are bringing to the platform, our roadmap, and listen to your feedback and suggestions.
+
+Agenda for this month's call, hosted by Matt Hidinger, included:
+
+Build 2020 and a look at the Roadmap
+------------------------------------
+
+Build 2020 was all-virtual this year, and we had the opportunity to share the latest and greatest coming to Adaptive Cards on the [YouTube session](https://aka.ms/m365sk134). The session focused on four areas, outlined below.
+
+### Adaptive Card Templating and Designer Updates
+
+Templating helps you save time by easily populating reusable card templates with data. [Release Candidate SDKs](https://docs.microsoft.com/en-us/adaptive-cards/templating/sdk) are now available for .NET and JavaScript, and we plan to make them Generally Available in the coming weeks. We also added rich data-binding support in the [Adaptive Card Designer](https://adaptivecards.io/designer) to make it even faster to design new cards based on your data payloads.
+
```

Comment:
[question] Are we referring to templating and data binding interchangeably at this point?

Reply:
It's a fair question. In this context I'm using data-binding to refer to the act of binding to data, which is what the Designer helps do with the little flyout menu that lets you pick a property to bind to. Templating generally refers to overall feature area, and includes the ability to write expressions, hide objects with `$when`, etc. Eventually I hope we can release an Expression Builder UI into the Designer (Like Power Automate has)